lib/routes.js

```diff
@@ -1,11 +1,49 @@
-const KoaRouter = require('koa-router')
+const router = require('koa-router')()
 const models = require('./models')
 
-const router = new KoaRouter()
-
 router.get('/', async (ctx) => {
-  await models.dbHeathCheck()
+  await models.dbHealthCheck()
   ctx.body = 'ok'
 })
 
+/* Team */
+
+router
+  .get('/team', async (ctx) => {
+    ctx.body = await models.db.team.findAll({order: 'name ASC'})
+  })
+  .get('/team/:id', async (ctx) => {
+    ctx.body = await models.db.team.findById(ctx.params.id)
+  })
+  .post('/team', async (ctx) => {
+    let name = ctx.request.body.name
+    await models.db.team
+      .findOrCreate({where: {name: name}})
+      .spread(function (team, created) {
+        if (!created) {
+          ctx.status = 409
+          ctx.body = `Team named "${name}" already exists!`
+        } else {
+          ctx.body = team
+        }
+      })
+  })
+  .put('/team/:id', async (ctx) => {
+    await models.db.team.update({
+      name: ctx.request.body.newname
+    }, {
+      where: {id: ctx.params.id}
+    })
+      .then(result => {
+        ctx.body = result
+      })
+      .catch(err => {
+        ctx.status = 409
+        ctx.body = err
+      })
+  })
+  .del('/team/:id', async (ctx) => {
+    ctx.body = await models.db.team.destroy({where: {id: ctx.params.id}})
```

Comment:
Created #15 and adding a TODO to address it. Currently, the status code returned will always be 200 but the body will be 0/1 depending on whether the operation failed/succeeded. Should I assign the result to a variable and use something like `ctx.status = isDeleted ? 204 : 400`? Also which codes should we use for deletion success vs. deletion failure?